department-of-veterans-affairs/va.gov-cms
# VAGOV-5038: Allow <ol> and <ul> in plain text.

QA: 
Go to /node/add/page and add `<ul><li>something</li><li>something else<li></ul><ol><li>something</li><li>something else<li></ol>` to the intro text. Fill out all other required fields with random stuff. Save node. Edit again. You should see your lists still there.
